src/components/footer/Footer.js

```diff
@@ -0,0 +1,19 @@
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <button>
```

Comment:
Please add an `aria-label` to the `button`s as this is not accessible.

Reply:
Ich habe hinzufügen